x/pairing/README.md

```diff
@@ -62,6 +62,8 @@ type StakeEntry struct {
 	DelegateTotal      types.Coin // total delegation to the provider (without self delegation)
 	DelegateLimit      types.Coin // delegation total limit
 	DelegateCommission uint64     // commission from delegation rewards
+	Jails              uint64     // number of times the provider have been jailed
+	JailTime           int64      // the time when the jail period is finished the he can come back to service
```

Comment:
the end of the jail sentence, after which the provider can return to service

Reply:
done

Reply:
https://github.com/lavanet/lava/pull/1443/commits/f1a0a0463a848c0605e28e3b068220f5afdf8ccb